activestorage/test/models/variant_test.rb

```diff
@@ -220,4 +246,15 @@ def process_variants_with(processor)
     ensure
       ActiveStorage.variant_processor = previous_processor
     end
+
+    def using_default_transformations(format, transformations)
```

Comment:
Let's mark the transformations as a keyword argument so we don't have to wrap the call-sites in `{}`.

```suggestion
    def using_default_transformations(format, **transformations)
```